src/rpc/blockchain.cpp

```diff
@@ -1583,15 +1583,15 @@ static UniValue invalidateblock(const JSONRPCRequest& request)
     uint256 hash(ParseHashV(request.params[0], "blockhash"));
     CValidationState state;
 
+    CBlockIndex* pblockindex;
     {
         LOCK(cs_main);
         CBlockIndex* pblockindex = LookupBlockIndex(hash);
```

Comment:
Commit 78122f3d7ed059b00bdaed585:

I think this is invalid cpp and wouldn't compile

Reply:
You mean should just be `pblockindex = LookupBlockIndex(hash);`?

Reply:
Yeah, I guess it compiles, but it might do the wrong thing.

Reply:
FIxed. This was fixed, but in a later commit.